app/bot.py

```diff
@@ -175,6 +176,22 @@ async def on_message(self, message: discord.Message):
 
         await self.process_commands(message)
 
+    async def on_member_join(self,member: discord.Member):
```

Comment:
Change function to 
`async def on_member_join(self, member: discord.Member):` 
(notice space between `self,` and `member`